sql/base.sql

```diff
@@ -1036,99 +1040,104 @@ INSERT OR IGNORE INTO RequirementArguments
+	('POLICY_TRENCH_WARFARE', 'LOC_POLICY_TRENCH_WARFARE_NAME' , 'LOC_POLICY_TRENCH_WARFARE_DESCRIPTION' , 'CIVIC_SCORCHED_EARTH' , 'SLOT_MILITARY');
 
+INSERT OR IGNORE INTO ObsoletePolicies
+	(PolicyType, ObsoletePolicy)
+	VALUES
+	('POLICY_SIEGE', 'POLICY_HARD_SHELL_EXPLOSIVES'),
+	('POLICY_HARD_SHELL_EXPLOSIVES', 'POLICY_TRENCH_WARFARE');
+
+INSERT OR IGNORE INTO PolicyModifiers
+	(PolicyType, ModifierId)
+	VALUES
+	('POLICY_SIEGE', 'SIEGE_ANCIENT_SIEGE_PRODUCTION'),
+	('POLICY_SIEGE', 'SIEGE_CLASSICAL_SIEGE_PRODUCTION'),
+	('POLICY_HARD_SHELL_EXPLOSIVES', 'HARD_SHELL_ANCIENT_SIEGE_PRODUCTION'),
+	('POLICY_HARD_SHELL_EXPLOSIVES', 'HARD_SHELL_CLASSICAL_SIEGE_PRODUCTION'),
+	('POLICY_HARD_SHELL_EXPLOSIVES', 'HARD_SHELL_MEDIEVAL_SIEGE_PRODUCTION'),
+	('POLICY_HARD_SHELL_EXPLOSIVES', 'HARD_SHELL_RENAISSANCE_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_ANCIENT_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_CLASSICAL_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_MEDIEVAL_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_RENAISSANCE_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_INDUSTRIAL_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_MODERN_SIEGE_PRODUCTION'),
+	('POLICY_TRENCH_WARFARE', 'TRENCH_WARFARE_ATOMIC_SIEGE_PRODUCTION');
+
+INSERT OR IGNORE INTO Modifiers
+	(ModifierId                 , ModifierType)
+	VALUES
+	('SIEGE_ANCIENT_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('SIEGE_CLASSICAL_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('HARD_SHELL_ANCIENT_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('HARD_SHELL_CLASSICAL_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('HARD_SHELL_MEDIEVAL_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('HARD_SHELL_RENAISSANCE_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_ANCIENT_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_CLASSICAL_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_MEDIEVAL_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_RENAISSANCE_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_INDUSTRIAL_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_MODERN_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION'),
+	('TRENCH_WARFARE_ATOMIC_SIEGE_PRODUCTION'         , 'MODIFIER_PLAYER_CITIES_ADJUST_UNIT_TAG_ERA_PRODUCTION');
+
+INSERT OR IGNORE INTO ModifierArguments
```

Comment:
removing the extra spaces between INSERTS would be appreciated, as well as the spaces before commas. not super important, just cosmetic.